code/salary-calculator/ViewModel/src/test/java/ru/unn/agile/salarycalculator/viewmodel/legacy/ViewModelTests.java

```diff
@@ -0,0 +1,160 @@
+package ru.unn.agile.salarycalculator.viewmodel.legacy;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import ru.unn.agile.salarycalculator.viewmodel.legacy.ViewModel.Status;
+
+import static org.junit.Assert.*;
+
+public class ViewModelTests {
+    private ViewModel viewModel;
+
+    @Before
+    public void setUpEmptyExample() {
+        viewModel = new ViewModel();
+        viewModel.setSalary("10000");
+        viewModel.setWorkedHours("154");
+        viewModel.setCountMonth("10");
+        viewModel.setCountYear("2014");
```

Comment:
If any of your "set" isn't working - all of your tests will be failed
I guess - you should move set block to anouther method.